Gemfile

```diff
@@ -54,18 +54,8 @@ group :test do
 end
 
 group :development, :test do
-  gem 'uglifier'
-
-  # Bootstrap is a css framework
-  # Pinning to v4 as bootstrap 5 drops compatibility with the latest versions of chrome and FF on XP
-  # Some lab machines are locked to XP due to vendor compatibility
```

Comment:
You've removed the comment saying why it's pinned, but not unpinned it. Understand if you don't want to deal with an upgrade at this time. Maybe you could just change the comment to say `it is pinned because of this, but doesn't need to be any more`